quantized_mesh_tile/utils.py

```diff
@@ -172,10 +169,9 @@ def computeNormals(vertices, faces):
 
     for i in xrange(0, numFaces):
         face = faces[i]
+        weightedNormal = [c * areasPerFace[i] for c in normalsPerFace[i]]
         for j in face:
-            weightedNormal = [c * areasPerFace[i] for c in normalsPerFace[i]]
-            normalsPerVertex[j] = c3d.add(
-                normalsPerVertex[j], weightedNormal)
+            normalsPerVertex[j] = c3d.add(normalsPerVertex[j], weightedNormal)
```

Comment:
If you feel like it I have a debugger here:
https://quantized-mesh-tile.readthedocs.io/en/latest/viewer.html?z=14&x=17066&y=12441&tileUrl=https%3A%2F%2Fterrain0.geo.admin.ch%2F1.0.0%2Fch.swisstopo.terrain.3d_light%2Fdefault%2F20151231%2F4326%2F14%2F17066%2F12441.terrain